### DISCUSSION TITLE
Direct LAN port?  Manager 1.57 works fine.

### DISCUSSION THREAD

**Discussion by C00K50N** [nathanmarlor/foxess_modbus#210] [General]
I haven't migrated over to this integration from the StealthChesnut one yet, because we connect via the inverter's direct LAN port.  We were on Manager version 1.44 for ages, then a couple months ago were updated to 1.57 and it's been absolutely fine, no issues with LAN connection before or since.

Did I read correctly when this integration was started that you weren't using the direct LAN connection at all?  Are there any plans to now or will I need to continue using the StealthChesnut integration instead?

**Comment by canton7** [comment]
A direct LAN connection is supported (and always has been), just the same as with StealthChesnut's integration

**Reply by C00K50N** [reply]
Ahhhh.... that's good news!  I've clearly misunderstood somewhere along the line.  Might be time to migrate over soon then.  Thank you.

**Reply by calum-mcfarlane** [reply]
The direct LAN connection doesn't give as much info as the RS485 one, nor does it allow writing of Modbus registers to control aspects of the inverter's operation (eg charge times). That said, if what you're doing works for you, no need to change it!

There was also a version of the Manager firmware that disabled the LAN port. Having gone from 1.44 to 1.57 you've skipped those versions though.
